packages/notifi-react/lib/context/NotifiTopicContext.tsx

```diff
@@ -128,7 +140,10 @@ export const NotifiTopicContextProvider: FC<PropsWithChildren> = ({
           fusionEventId: fusionEventId,
           name: customAlertName ?? topic.uiConfig.name,
           targetGroupId,
-          subscriptionValue: subscriptionValue ?? legacySubscriptionValue,
+          subscriptionValue:
+            subscriptionValue ??
+            subscriptionValueObject?.[0]?.value ??
+            legacySubscriptionValue,
```

Comment:
The suggestion continued from above.
```suggestion
            subscriptionValue: subscriptionValue ?? legacySubscriptionValue,
```